great_expectations/core/validation.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from great_expectations.core.batch_config import BatchConfig
+    from great_expectations.core.expectation_suite import ExpectationSuite
+    from great_expectations.datasource.fluent.interfaces import DataAsset
+
+
+class Validation:
```

Comment:
Do we want this to be a Pydantic model? Went back and forth on it but figured its a pretty lightweight object so we can just add it if we need (probably when persistence comes into question?)